crt_portal/cts_forms/models.py

```diff
@@ -349,6 +349,11 @@ def related_reports_display(self):
                 ('closed', reports.filter(status='closed')),
                 )
 
+    @property
+    def contact_full_name(self):
+        full_name = f'{self.contact_first_name}, {self.contact_last_name}'
```

Comment:
If there's no last name, this will look odd. Likewise if there's no first name. And of course there can be no first or last name given. There's some common code with the `addressee` property that we can perhaps refactor/reuse. (Perhaps `addressee` should use a modified version of `contact_full_name` instead)

Reply:
Maybe:

```python
    @property
    def contact_full_name(self):
        if self.contact_first_name:
            if self.contact_last_name:
                return f"{self.contact_first_name} {self.contact_last_name}"
            return f"{self.contact_first_name}"
        return "Anonymous"
```

(Also, there shouldn't be a comma between first and last name)

Reply:
Oof yeah, my version -- cribbed from Django itself -- doesn't work here. We're storing both `None` and empty strings for `contact_first_name` and `contact_last_name`. Thanks @cantsin! I'll rework this.

Reply:
Added https://github.com/usdoj-crt/crt-portal/pull/793/commits/d49af0b7aec51063f2e74a3e437d6e5cb8d08162 !

Reply:
I left `Anonymous` out to stay consistent with how we're rendering non-present names elsewhere.